align PFD 18 and 28 seconds into alignment

AdenFlorian opened the pull request:
closes #147 

Ties PFD FAIL elements to the new local var L:A32NX_ADIRS_PFD_ALIGNED, so that PFD won't become available until 40% through alignment.

NathanInnes commented on A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/A320_Neo_PFD.html:
```diff
@@ -76,14 +76,14 @@
+<script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/AirspeedIndicator.js?rrr=1"></script> <!-- MODIFIED -->
+<script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/AltimeterIndicator.js?jjj=1"></script> <!-- MODIFIED -->
 <script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/Shared/PFD/HSIndicator.js"></script>
 <script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/AttitudeIndicator.js"></script> <!-- MODIFIED -->
-<script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/VerticalSpeedIndicator.js"></script> <!-- MODIFIED -->
-<script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/ILSIndicator.js"></script> <!-- MODIFIED -->
+<script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/VerticalSpeedIndicator.js?lll=1"></script> <!-- MODIFIED -->
+<script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/ILSIndicator.js?eee=1"></script> <!-- MODIFIED -->
 <script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/Airbus_FMA.js"></script> <!-- MODIFIED -->
 
 <script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/Shared/BaseAirliners.js"></script>
 
-<script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/A320_Neo_PFD.js"></script>
+<script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/A320_Neo_PFD.js?ppp=1"></script>
```
Same here

NathanInnes replied:
Suggest changing here also

NathanInnes commented on A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU.html:
```diff
@@ -61,4 +61,4 @@
 <script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_NewWaypoint.js"></script> 
 <script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_PilotsWaypoint.js"></script> 
 
-<script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js"></script>
```
Please remove the ?xxx=xx from the end as it's not needed for build.

NathanInnes commented on A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU.html:
```diff
@@ -61,4 +61,4 @@
 <script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_NewWaypoint.js"></script> 
 <script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_PilotsWaypoint.js"></script> 
 
-<script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js"></script>
+<script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js?ttt=1"></script>
```
Suggest removing ?xxx=xxx at the end as it's not needed for build.

NathanInnes commented on A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU.html:
```diff
@@ -61,4 +61,4 @@
 <script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_NewWaypoint.js"></script> 
 <script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_PilotsWaypoint.js"></script> 
 
-<script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js"></script>
+<script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js?ttt=1"></script>
```
Suggest removing the ?xxx=xxx at the end as its not needed for dev build.

AdenFlorian replied:
oops, didn't mean to commit those, fixed!

NathanInnes commented on A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/A320_Neo_PFD.html:
```diff
@@ -76,14 +76,14 @@
+<script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/AirspeedIndicator.js?rrr=1"></script> <!-- MODIFIED -->
+<script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/AltimeterIndicator.js?jjj=1"></script> <!-- MODIFIED -->
 <script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/Shared/PFD/HSIndicator.js"></script>
 <script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/AttitudeIndicator.js"></script> <!-- MODIFIED -->
-<script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/VerticalSpeedIndicator.js"></script> <!-- MODIFIED -->
-<script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/ILSIndicator.js"></script> <!-- MODIFIED -->
+<script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/VerticalSpeedIndicator.js?lll=1"></script> <!-- MODIFIED -->
+<script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/ILSIndicator.js?eee=1"></script> <!-- MODIFIED -->
 <script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/Airbus_FMA.js"></script> <!-- MODIFIED -->
 
 <script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/Shared/BaseAirliners.js"></script>
 
-<script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/A320_Neo_PFD.js"></script>
+<script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/A320_Neo_PFD.js?ppp=1"></script>
```
Suggested change here also.

AdenFlorian replied:
fixed!

AdenFlorian commented:
> I think the airspeed and altitude indicators are supposed to start working as soon as the adirs are set to nav.

I'm asking around.

AdenFlorian commented:
Changed it so the airspeed, altimeter and vertical speed become available 18 seconds after alignment start, and the attitude indicator becomes available 28 seconds after alignment start. These timings come from a video.

Also fixed it so the target altitude cyan stuff isnt visible until the whole altimeter is visible.

before and beginning of alignment:
![image](https://user-images.githubusercontent.com/3533988/91931848-340bb780-ecaa-11ea-8027-5929edf96c27.png)


18 seconds in:
![FlightSimulator_1599017623](https://user-images.githubusercontent.com/3533988/91929994-4d5e3500-eca5-11ea-83d4-98a2d719cb6a.png)

28 seconds in:
![FlightSimulator_1599017633](https://user-images.githubusercontent.com/3533988/91929997-50f1bc00-eca5-11ea-93a2-80ea2e926d9f.png)

what it looks like after alignment is finished is the same as before.
![image](https://user-images.githubusercontent.com/3533988/91930584-d6c23700-eca6-11ea-84d3-d05535381516.png)

AdenFlorian commented:
done testing, let me know if anything else is needed!
